README.md

```diff
@@ -60,18 +60,19 @@ To start using `dffml` for machine learning with a few CLI commands see
 
 Start with [Architecture](docs/ARCHITECTURE.md).
 
-## Tutorials
-
-Tutorials will get you writing code that takes full advantage of the DFFML API.
-Making you're next machine learning project a breeze to write!
-
-- Features
-  - The [new feature tutorial](docs/tutorial/FEATURE.md) will walk you through
-    how to write a new DFFML feature to generate data for a dataset.
-- Models
-  - The [new model tutorial](docs/tutorial/MODEL.md) will walk you through how
-    to wrap your favorite framework or a custom implementation in the DFFML
-    library's model API.
+## Contributing
```

Comment:
Please leave the Tutorials section and add the Contributing section above it.

Reply:
Should I make them nested of tutorials should be separate?